### PR TITLE
Bug 1628364 - WNPanel messages for 76 + archive of older messages

### PR DESCRIPTION
**Pull request by piatra**

### CODE REVIEW

File: schema/whats-new-panel.schema.json [hunk 1]
```diff
@@ -8,6 +8,11 @@
       "description": "Plain text (no HTML allowed)",
       "type": "string"
     },
+    "validStringOrID": {
+      "type": "string",
+      "description": "General purpose valid text with verification",
+      "pattern": "^[A-Za-z0-9-_]*$"
```
Comment (piatra) [inline schema/whats-new-panel.schema.json:14]: I was thinking we could replace "string" with this definition+pattern check. It's not meant for validation but more for catching mistakes like the "," and so on.

Comment (ncloudioj) [reply schema/whats-new-panel.schema.json:14]: This is great, I like it.

File: schema/whats-new-panel.schema.json [hunk 2]
```diff
@@ -35,41 +41,26 @@
...
-              "properties": {
-                "string_id": {
-                  "type": "string",
-                  "description": "Id of string to localized addon description"
-                }
-              },
-              "required": ["string_id"]
-            }
-          ]
+          "$ref": "#/definitions/stringOrStringID"
```
Comment (ncloudioj) [inline schema/whats-new-panel.schema.json:56]: Hmm, what if `body` is an object with a `string_id`? Perhaps you wanted to just replace the arm for the `string` type?

File: whats-new-panel.yaml [hunk 3]
```diff
@@ -260,3 +98,63 @@
...
+    body:
+      string_id: cfr-whatsnew-pip-fullscreen-body
+    cta_url: https://support.mozilla.org/1/firefox/%VERSION%/%OS%/%LOCALE%/about-p2p
+    cta_type: OPEN_URL
+    link_text:
+      string_id: cfr-whatsnew-pip-cta
+  targeting: firefoxVersion >= 76
+  trigger:
+    id: whatsNewPanelOpened
+- id: WHATS_NEW_VULNERABLE_PASSWORDS
```
Comment (ncloudioj) [inline whats-new-panel.yaml:121]: By convention, let's append a version suffix to the `id` and `bucket_id` fields for vulnerable_passwords and lockwise_passwords.

File: whats-new-panel.yaml [hunk 4]
```diff
@@ -260,3 +98,63 @@
...
+  template: whatsnew_panel_message
+  order: 2
+  content:
+    bucket_id: WHATS_NEW_VULNERABLE_PASSWORDS
+    published_date: 1586859903200
```
Comment (ncloudioj) [inline whats-new-panel.yaml:126]: Shall we just set this to the release date timestamp (1588636800000 i.e. 5th May 2020) for 76?

File: whats-new-panel.yaml [hunk 5]
```diff
@@ -260,3 +98,63 @@
...
+      string_id: cfr-whatsnew-pip-cta
+  targeting: firefoxVersion >= 76
+  trigger:
+    id: whatsNewPanelOpened
+- id: WHATS_NEW_LOCKWISE_PASSWORDS
+  template: whatsnew_panel_message
+  order: 1
+  content:
+    bucket_id: WHATS_NEW_LOCKWISE_PASSWORDS
+    published_date: 1586859903200
```
Comment (ncloudioj) [inline whats-new-panel.yaml:146]: Ditto.

File: whats-new-panel.yaml [hunk 6]
```diff
@@ -260,3 +98,63 @@
   template: toolbar_badge
   trigger:
     id: toolbarBadgeUpdate
+- id: WHATS_NEW_PIP_76
+  template: whatsnew_panel_message
+  order: 3
+  content:
+    bucket_id: WHATS_NEW_PIP_76
+    published_date: 1586859903200
```
Comment (ncloudioj) [inline whats-new-panel.yaml:106]: Is this a new PIP message? If not, shall we stick with the `published_date` of the current PIP?

Comment (piatra) [reply whats-new-panel.yaml:106]: New message, it will replace the old one but we can't switch until the release day.

Comment (piatra) [reply whats-new-panel.yaml:106]: Or actually I can add version targeting to the old one.

File: whats-new-panel.yaml [hunk 7]
```diff
@@ -260,3 +98,63 @@
...
+    body:
+      string_id: cfr-whatsnew-passwords-body
+    cta_url: https://support.mozilla.org/1/firefox/%VERSION%/%OS%/%LOCALE%/lockwise-alerts
+    cta_type: OPEN_URL
+    link_text:
+      string_id: cfr-whatsnew-pip-cta
+  targeting: firefoxVersion >= 76
+  trigger:
+    id: whatsNewPanelOpened
+- id: WHATS_NEW_LOCKWISE_PASSWORDS
```
Comment (ncloudioj) [inline whats-new-panel.yaml:141]: Let's add a version to this as well.

**Pull request merged**